Softeq.XToolkit.ProjectInfrastructure/Executor/Executor.cs

```diff
@@ -22,7 +22,10 @@ static Executor()
         /// <param name="allowedAttempts">Show how many times action will be executed again in case of exception</param>
         /// <param name="executionGroup">Indicates the group of actions to which current execution belongs. If something fails in group then notification will appear only once for all actions in group. Use null if action is group independent</param>
         /// <returns></returns>
-        public static async Task ExecuteWithRetryAsync(Func<IAsyncExecutionContext, Task> asyncAction, int allowedAttempts = 1, string executionGroup = null)
+        public static async Task ExecuteWithRetryAsync(Func<IAsyncExecutionContext, Task> asyncAction,
```

Comment:
Do we need to change Executor, as long as we agreed not to use it within rest services?

Reply:
Oh, actually no. But I'd like to leave these changes - code became a bit cleaner and exceptionHandler can be used